src/main/java/com/google/cloud/tools/maven/DeployQueueMojo.java

```diff
@@ -20,7 +20,7 @@
 import org.apache.maven.plugins.annotations.LifecyclePhase;
 import org.apache.maven.plugins.annotations.Mojo;
 
-/** Stage and deploy queue.yaml to Google App Engine standard or flexible environment. */
+/** Stage and deploy queue.yaml to Google App Engine. */
```

Comment:
I think you'll need to update this comment on all the `DeployXMojo`s

Reply:
done